dubbo-configcenter/dubbo-configcenter-api/src/main/java/org/apache/dubbo/configcenter/DynamicConfiguration.java

```diff
@@ -104,6 +111,22 @@ default String getConfig(String key, String group) {
      */
     String getConfig(String key, String group, long timeout) throws IllegalStateException;
 
+    /**
+     * {@see #getConfig(String, String, long)}
+     *
+     * This method are mostly used to get a compound config file, such as a complete dubbo.properties file.
+     */
+    default String getConfigFile(String key, String group) throws IllegalStateException {
```

Comment:
suggest change to: 

```java
    default String getProperties(String group) {
        return getProperties(group, 0L);
    }
    
    String getProperties(String group, long timeout);
```

Reply:
Do you mean to use a fixed key `dubbo.properties`?  But we have given users the  to provide a customized key by `ConfigCenterConfig.configFile`: https://github.com/apache/incubator-dubbo/blob/9f3ac611eff6f3918fd9e367a90f76d0ee9cc609/dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ConfigCenterConfig.java#L47

We should remove `ConfigCenterConfig.configFile` at the same time if decide to remove key from this interface?

Reply:
> suggest change to:
> 
>     default String getProperties(String group) {
>         return getProperties(group, 0L);
>     }
>     
>     String getProperties(String group, long timeout);

1. I have changed to `String getConfigs(String key, String group, long timeout);`, in case of supporting `yaml` format in the future, `getProperties` would be confusing.
2. I think it would be more flexible to have the `key` parameter kept.